data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,58 @@ mission "Remnant: Shattered Light 6"
 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
 			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`
+
+
+
+mission "Remnant: Scent of Rain 1"
+	landing
+	name "Search for the unknown Korath ship."
+	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
+	source
+		government "Remnant"
```

Comment:
This could offer on Ssil Vida but the conversation seems to assume we're not there.
`I read a report mentioning that you dealt with a trio of 'olofez out in the Postverta region.`

Reply:
Hmmmm, I think that's fine to say. Someone in California could tell someone else "So, I hear you have been starting businesses in the California region. Might be slightly odd turn of phrase, but no more so than, say, someone speaking in the third person or something. By which I mean, perfectly acceptable but different.